src/rollback_to_stable/rts_api.c

```diff
@@ -144,9 +150,11 @@ __rollback_to_stable_one(WT_SESSION_IMPL *session, const char *uri, bool *skipp)
     F_CLR(session, WT_SESSION_QUIET_CORRUPT_FILE);
 
     __rts_assert_timestamps_unchanged(session, pinned_timestamp, rollback_timestamp);
+    __wt_timer_evaluate(session, &timer, &time_diff);
+    __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
+      "finished rollback to stable on uri %s and has ran for %" PRIu64 " seconds", uri, time_diff);
```

Comment:
Same change of seconds to milliseconds here.

Reply:
done [43412ee](https://github.com/wiredtiger/wiredtiger/pull/9710/commits/43412ee632cb94a4c50325a35b443538e193b6bf)